File: Python/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        Dict = {}
        for i in range(len(nums)):
            complement = target - nums[i]
            if nums[i] in Dict:
                return [Dict[nums[i]], i]
            else:
                Dict[complement] = i
        return None

File: Java/1. Two Sum.java
import java.util.HashMap;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> map = new HashMap<>();
        int complement;
        for(int i = 0; i < nums.length; i++){
            complement = target - nums[i];
             if(map.containsKey(nums[i])){
                return new int[] {map.get(nums[i]), i};
             }
             else{
                map.put(complement,i);
             }
        }
        return null;
    }
}